src/sse/route-service.ts

```diff
@@ -69,20 +69,26 @@ export default class RouteService {
     // TODO handle limit intelligently - wait if we haven't reached that yet, etc
     // this is needed regardless to refresh initial data send to the latest data
     // TODO handle different permanence parameters, eg ?permanence=confirmed or =all 
-    this.sse.updateInit(
-      this.eventService.state.getAllEvents({ limit, minHeight }),
-    );
+    const initialEvent: InitialEvent = {
+      config: {
+        network,
+        type: this.type,
+        id: this.filter,
+        maxConf: confirmationDepth,
+        heartbeat: heartbeatInterval,
+        v:  '0.0.2',
```

Comment:
I was trying to keep the strings short as these will be going over the wire - hence `maxConf` instead of `confirmationDepth`, shortened `heartbeatInterval` etc